packages/client/src/intros/shared/Intro.tsx

```diff
@@ -43,14 +45,18 @@ export default ({
 
   return (
     <>
-      <Paragraph gutterBottom={usableForBullets.length > 0 ? 8 : undefined}>
-        {t("introPage.common.check for permit intro")}{" "}
-        {usableForBullets.length > 0 && t("introPage.common.permit for")}
+      { needReport && <Paragraph>
+        {t("introPage.demolition.need report")}{" "}
+      </Paragraph> }
+
+      <Paragraph gutterBottom={usableForBullets?.length > 0 ? 8 : undefined}>
+        {!needReport && t("introPage.common.check for permit intro")}{" "}
```

Comment:
Why don't we show this text in case the outcomes include needReport?
It can still be a combination of both right?